api-specificatie/zrc/openapi.yaml

```diff
@@ -72,83 +75,95 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Fout'
+    parameters:
+      - name: version
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: zaakid
+        in: path
+        required: true
+        description: zaakidentificatie
+        schema:
+          type: string
+servers:
+  - url: /
 components:
   schemas:
     Zaak:
-      description: Een zaak
+      description: Een ZAAK
+      required:
+        - zaaktype
+      type: object
       properties:
-        type:
-          description: URL naar het zaaktype van de zaak
-          type: string
-        omschrijving:
-          description: Omschrijving of toelichting op de zaak
-          type: string
-        initiator:
-          $ref: '#/components/schemas/Subject'
+        url:
+          title: Url
+          type: string
+          format: uri
+          readOnly: true
+        zaakidentificatie:
+          title: Zaakidentificatie
+          description: >-
+            De unieke identificatie van de ZAAK binnen de organisatie die
+            verantwoordelijk is voor de behandeling van de ZAAK.
+          type: string
+          maxLength: 40
+        zaaktype:
+          title: Zaaktype
+          description: URL naar het zaaktype in de CATALOGUS waar deze voorkomt
+          type: string
+          format: uri
+          maxLength: 200
+          minLength: 1
         status:
-          description: URL naar de status van de zaak
+          title: Status
+          description: 'Indien geen status bekend is, dan is de waarde ''null'''
           type: string
+          format: uri
+          readOnly: true
+        omschrijving:
+          title: Omschrijving
+          description: Een korte omschrijving van de zaak.
+          type: string
+          maxLength: 80
+        # TODO: in het RGBZ 2.0 model loopt dit via de rollen/betrokkenen
+        # initiator:
+        #   title: Initiator
+        #   description: URL naar de initiator van de ZAAK.
+        #   type: string
+        #   format: url
         registratiedatum:
-          description: Tijdstip waarop de zaak is geregistreerd
-          type: string
-          format: 'date-time'
-        kanaal:
-          description: Kanaal via welke de zaak is gemeld
-          type: string
-        locatie:
-          description: Coordinate van de locatie waarop de zaak betrekking heeft in Well Known Text (WKT) formaat
+          title: Registratiedatum
+          description: De datum waarop de zaakbehandelende organisatie de ZAAK heeft geregistreerd
+          type: string
+          format: 'date'
+        # TODO: volgens RGBZ 2.0 komt kanaal uit KLANTCONTACT
+        # kanaal:
+        #   description: Kanaal via welke de zaak is gemeld
+        #   type: string
+        zaakgeometrie:  # NOTE: komt ook voor bij OBJECT
```

Comment:
Een Zaak is *ook* een Object 🥁, vandaar.

Reply:
Ik ben abuis, Zaak erft niet van Object. Er bestaat wel zoiets als ZaakObject.... ongerelateerd.